### PR TITLE
iboss - Add policy layer list management

### PR DESCRIPTION
**Pull request by mvforgione**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Add policy later list manage (remove/add entries)
- Add reliability config parameter

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [x] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by mvforgione** [comment]
@yaakovi will you be performing the demisto/pull-check "Awaiting documentation review" step or is there something else I must do to kick that off?

**Review by yaakovi — CHANGES_REQUESTED** [review]
Revert the changes in the `package-lock.json` file

File: Packs/Iboss/Integrations/Iboss/Iboss.py [hunk 1]
```diff
@@ -1498,6 +1639,154 @@ def remove_entity_from_allow_list_command(client: Client, args: Dict[str, Any])
...
+                start_port=start_port, end_port=end_port, direction=direction,
+                do_dlp_scan=do_dlp_scan,
+                do_malware_scan=do_malware_scan, note=note, priority=priority,
+                time_url_expires_in_seconds=time_url_expires_in_seconds,
+                is_regex=is_regex)
+
+            if 'Successfully' in result['message']:
+                result['message'] = f'Successfully updated {url} in policy layer `{policy_layer_name}`.'
+
+        command_results.append(CommandResults(
```
[the remark this answers is not in the document]
Comment (mvforgione) [reply Packs/Iboss/Integrations/Iboss/Iboss.py:1719]: @yaakovi originally the result output was patterned around other partner solutions that used similar methodology. This is also the same output format as the pre-existing list modification methods. Is it better to update these new methods with the table view?

**Comment by mvforgione** [comment]
> Revert the changes in the `package-lock.json` file

@yaakovi just so I understand, the package-lock.json file change is upping the current version to 1.0.3. Is that not appropriate given that is the current version for this PR?

**Comment by yaakovi** [comment]
> > Revert the changes in the `package-lock.json` file
> 
> @yaakovi just so I understand, the package-lock.json file change is upping the current version to 1.0.3. Is that not appropriate given that is the current version for this PR?

Need to revert the `package-lock.json` changes, not the `pack_metadata.json`.
* pack_metadata.json - the metadata of the pack including the pack version.
* package-lock.json - build related version of node packages

File: Packs/Iboss/Integrations/Iboss/Iboss.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaakovi) [reply Packs/Iboss/Integrations/Iboss/Iboss.py:1719]: I think a single table has better visibility over many tables. With the overhead between tables, you will get less data on the screen.
Also, the user can expand a single entry in a new tab for complete visibility.

**Review by yaakovi — COMMENTED** [review]
see my responses on the previous comments

**Comment by mvforgione** [comment]
> see my responses on the previous comments

comments addressed

File: Packs/Iboss/Integrations/Iboss/Iboss.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mvforgione) [reply Packs/Iboss/Integrations/Iboss/Iboss.py]: Table views added for results for policy layer lists

**Comment by mvforgione** [comment]
@yaakovi can you let me know what time you will be able to review the changes? I will perform a merge from upstream master at that time (currently I'm continuously performing this operation)

**Pull request merged**